tools/BatteryModel.py

```diff
@@ -0,0 +1,42 @@
+# Class model of a standard LiPo battery
+
+import sys
+import pyulog as ulog
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.optimize import curve_fit
+
+class BatteryModel:
+    def __init__(self, file, nominal_voltage: float = 11.1, nominal_mAh: float = 5100.0, cells: int = 3):
+   
+        # Givens
+        # self.__nominal_voltage = nominal_voltage
+        self.__cells = cells                      
+        self.__nominal_Ah = nominal_mAh/1000.0
+        
+        self.__nominal_capacity = 8500      # mAh
```

Comment:
never hardcode magic numbers like this. It should be an input to the constructor (which already exists)

Reply:
> this is just a note since we are just removing `__init__`